.github/workflows/pit.yml

```diff
@@ -88,6 +88,8 @@ jobs:
           M=`echo $M | sed -e s/,$//`']}'
           echo "Generated Matrix: $M"
           echo "matrix=$M" >> $GITHUB_OUTPUT
+        env:
+          CE_LICENSE: ${{secrets.CE_LICENSE}}
```

Comment:
do we still need the CE_license ? CE is a free product now.

Reply:
Just for testing old branches, we can remove in a future